workorder/customizable_workorder.tpl

```diff
@@ -290,7 +290,11 @@ img.barcode {
 				<p id="customerName">{{ Workorder.Customer.firstName}} {{ Workorder.Customer.lastName}}</p>
 				<p id="customerAddress1">{{ Workorder.Customer.Contact.Addresses.ContactAddress.address1 }}</p>
 				<p id="customerAddress2">{{ Workorder.Customer.Contact.Addresses.ContactAddress.address2 }}</p>
-				<p id="customerAddressCity">{{ Workorder.Customer.Contact.Addresses.ContactAddress.city }}, {{ Workorder.Customer.Contact.Addresses.ContactAddress.state }} {{ Workorder.Customer.Contact.Addresses.ContactAddress.zip }}</p>
+				<p id="customerAddressCity">
+          {{ Workorder.Customer.Contact.Addresses.ContactAddress.city }}{% if Workorder.Customer.Contact.Addresses.ContactAddress.city and (Workorder.Customer.Contact.Addresses.ContactAddress.state or Workorder.Customer.Contact.Addresses.ContactAddress.zip) %},{% endif %}
+          {{ Workorder.Customer.Contact.Addresses.ContactAddress.state }} 
+          {{ Workorder.Customer.Contact.Addresses.ContactAddress.zip }}

```

Comment:
if empty, will not crash, right ?

Reply:
if there's no address info at all you mean?

Reply:
and no, it won't :D I added a bunch of screenshots from my testing [here](https://jira.atlightspeed.net/browse/LSR-13879)

Reply:
if city, state or zip is not set, it just doesn't display anything, right ?
(VS crashing because "variable blabla.state is undefined")

Reply:
that's an expensive Dyson vacuum 😮

Reply:
I mean, they practically cost an arm and a leg!